bears/general/QuotesBear.py

```diff
@@ -56,7 +56,8 @@ def correct_single_line_str(self, filename, file, sourcerange,
         diff.change_line(sourcerange.start.line,
                          file[sourcerange.start.line - 1],
                          replacement)
-        yield Result(self, 'You do not use the preferred quotation marks.',
+        yield Result(self, 'You do not use the preferred quotation marks.'
+                     + 'Preferred mark: ' + preferred_quotation,
```

Comment:
I think one space should be better before `Preferred` as there is no space after 1st sentence. I mean `'Preferred mark: '`--> `' Preferred mark: '`.